cmd/core-data/res/configuration.toml

```diff
@@ -1,3 +1,5 @@
+MaxEventSize = 25000  # Defines the maximum event size in kilobytes
```

Comment:
Why are we adding a new setting when we already have Service.MaxRequestSize, which is currently not used. Are you trying to implement this issue?
https://github.com/edgexfoundry/edgex-go/issues/3612#issuecomment-1057731152

Reply:
@lenny-intel we had the same conversation before.  please see https://github.com/edgexfoundry/edgex-go/issues/3237#issuecomment-926760249